docs/cloud-native-application-developers-manual.md

```diff
@@ -0,0 +1,60 @@
+# Cloud Native Application Developers Manual
+
+This document is intended for cloud native application authors targeting application deployments on Bitnami Kubernetes Production Runtime (BKPR). It describes the configuration an application author needs to perform for automating the Ingress, TLS, logging and monitoring support for their application.
+
+## Introduction
+
+BKPR is a curated collection of services running on top of your existing Kubernetes cluster with the aim of automating the configuration of public access, logging and monitoring, and management of DNS records and TLS certificates.
+
+For BKPR to perform this automation application authors need to apply a few configurations in their application manifests. The following sections walk you through each of these configurations.
+
+## Ingress and Let's Encrypt
+
+BKPR installs a [nginx-ingress](HTTPS://github.com/kubernetes/ingress-nginx) controller to the cluster which provides load balancing, SSL termination and name-based virtual hosting for your applications. Ingress enables application developers to expose HTTP and HTTPS routes from outside the cluster to services running inside the cluster.
```

Comment:
There is an inconsistency in how you refer to the reader. In some places, you say "your" and "you" eg. "your applications". In others, you refer to the reader in the third person eg. the "application author" or "application developer". Could you check and make this consistent?